src/domains/observability/.terraform.lock.hcl

```diff
@@ -1,6 +1,25 @@
 # This file is maintained automatically by "terraform init".
 # Manual edits may be lost in future updates.
 
+provider "registry.terraform.io/azure/azapi" {
+  version = "2.0.1"
+  hashes = [
+    "h1:NybiPNJ4MsKsTtRUnxV5vrhkbPkK+puqgfa0j0Q5nlU=",
```

Comment:
required?

Reply:
I don't know if it is required, it's a change made by an apply/plan

Reply:
revert this change, it is a file from a different domain  not related to the PR (perhaps you need to review the terraform configurations or your local configurations)

Reply:
Done in [f76c1ae](https://github.com/pagopa/pagopa-infra/pull/2504/commits/f76c1ae1639308e19e1f4bdd9ae1fad39cb4bf03)